### PR TITLE
Use babel for module path replacement

### PR DESCRIPTION
**Pull request by yesmeck**
See https://github.com/ant-design/ant-design/issues/6310#issuecomment-305176273 and https://github.com/ant-design/ant-design/commit/0080b6ddf5bae47531f3429d4680c37d45513d37#commitcomment-22345793

### CODE REVIEW

File: lib/getBabelCommonConfig.js [hunk 1]
```diff
@@ -8,6 +10,9 @@ module.exports = function (modules) {
   if (modules !== false) {
     plugins.push(require.resolve('babel-plugin-add-module-exports'));
   }
+  if (modules === false) {
+    plugins.push(replaceLib);
```
Comment (paranoidjk) [inline lib/getBabelCommonConfig.js]: 这个 plugin 必须在 babel-plugin-import 之后吧？ https://github.com/ant-design/antd-tools/blob/b8d709a03b9b1710cd5b0dfd1d766302ad97007b/lib/getWebpackConfig.js#L25

Comment (yesmeck) [inline lib/getBabelCommonConfig.js]: 是的。

**Comment by yesmeck** [comment]
@benjycui

File: lib/getBabelCommonConfig.js (hunk 1 above)
Comment (paranoidjk) [inline lib/getBabelCommonConfig.js]: 那这里就需要改吧

**Comment by yesmeck** [comment]
已改

**Comment by paranoidjk** [comment]
还是有问题，https://github.com/yesmeck/antd-tools/blob/f34c0590b070a4bb6d14e51297c3c485f919c75a/lib/gulpfile.js#L135 这样会导致 replacePlugin 添加两遍

**Comment by yesmeck** [comment]
哪里加了两遍？

**Comment by paranoidjk** [comment]
哦，看错了，已经改掉了

**Comment by paranoidjk** [comment]
@yesmeck  Added. need to fix this before next release.
```bash
npm owner ls antd-tools                                                 
yesmeck <yesmeck@gmail.com>
afc163 <afc163@gmail.com>
benjycui <benjytrys@gmail.com>
warmhug <hualei5280@gmail.com>
paranoidjk <hust2012jiangkai@gmail.com>
yiminghe <yiminghe@gmail.com>
```

**Comment by paranoidjk** [comment]
make sure  use npm link to test at local first

**Comment by benjycui** [comment]
👍

**Comment by benjycui** [comment]
antd-tools 之前 link 测试好像有问题，还是要先发布 @warmhug 

我发个 minor， @paranoidjk 把 https://github.com/ant-design/ant-design-mobile/blob/master/package.json#L75 改为 `~` 吧

**Comment by yesmeck** [comment]
用 yarn 可以 link 的。。。

**Comment by paranoidjk** [comment]
@benjycui link 的问题已经修复了。

**Comment by benjycui** [comment]
also fixed https://github.com/ant-design/antd-tools/commit/1121e028f1760dfe0866cf2c318e3860921583f1#commitcomment-22304256

**Comment by benjycui** [comment]
本地 compile 正常 👍

**Pull request merged**